packages/bootstrap/scss/index.scss

```diff
@@ -409,6 +411,7 @@
     @include kendo-splitter--styles();
     @include kendo-tile-layout--styles();
     @include kendo-grid-layout--styles();
+    @include kendo-stack-layout--styles();
```

Comment:
In my opinion we should name the mixin kendo-stacklayout--styles() (as the component is stacklayout)